aldryn_news/views.py

```diff
@@ -31,6 +31,16 @@ class ArchiveView(BaseNewsView, ArchiveIndexView):
     template_name = 'aldryn_news/news_list.html'
     date_list_period = 'month'
 
+    @property
+    def uses_datetime_field(self):
+        """Return False.
```

Comment:
If we know that this is version specific, it'd be good to add a check (something like https://github.com/aldryn/aldryn-jobs/blob/9be4193578b4a2959ea86294ea0872204ae4ce4c/aldryn_jobs/models.py#L129)